### ISSUE TITLE
No offset found for Asia/Tokyo timezone

### ISSUE THREAD

**Issue by xlegs** [SpiritIT/timezonecomplete#22]
When doing tc.add, Asia/Tokyo yields an error. My code works the same for other time zones, with all other values remaining constant.

EDIT: changed diff to add

**Comment by rogierschouten** [comment]
Hi @xlegs.  Thank you for entering an issue. Which version of timezonecomplete are you using? Can you send some code that shows the problem? I tried to reproduce this, but it seems to work OK:

```
var tc = require("timezonecomplete");
var d1 = new tc.DateTime("2016-02-01T08:00:00.000 Asia/Tokyo");
var d2 = new tc.DateTime("2016-02-01T09:00:00.000 Asia/Tokyo");
d1.diff(d2).toString(); // '-3600000 milliseconds'
```

**Comment by xlegs** [comment]
Looks like the error is actually in add. Here is my code. The var values were confirmed by debugger.

```
// 2016, 2, 12, 13, 0, 0, 0, timezonecomplete.zone("America/Los_Angeles")
var deptTime = new timezonecomplete.DateTime(d.deptTime.year, d.deptTime.month, d.deptTime.day, d.deptTime.hour, d.deptTime.minute, 0, 0, timezonecomplete.zone(d.deptTime.tzCode));

// 2016, 2, 12, 11, 0, 0, 0, timezonecomplete.zone("Asia/Tokyo")
var arrivalTime = new timezonecomplete.DateTime(d.deptTime.year, d.deptTime.month, d.deptTime.day, a.arrivalTime.hour, a.arrivalTime.minute, 0, 0, timezonecomplete.zone(a.arrivalTime.tzCode));

// days = 1; Error occurs here
arrivalTime = arrivalTime.add(timezonecomplete.days(days));

return arrivalTime.format("EEEE, MMMM d, YYYY").toString();
```

Running v1.27.1

**Comment by rogierschouten** [comment]
You are right:

The example:
```javascript
var tc = require("timezonecomplete");
var arrivalTime = new tc.DateTime(2016, 2, 12, 11, 0, 0, 0, tc.zone("Asia/Tokyo"));
arrivalTime = arrivalTime.add(tc.days(1));
```

The error:
```
 Error: No offset found.
      at TzDatabase.dstOffsetForRule (tz-database.ts:935:10)
      at TzDatabase.totalOffset (tz-database.ts:786:22)
      at TimeZone.offsetForUtc (timezone.ts:372:35)
      at DateTime._utcDateToZoneDate (datetime.ts:1143:36)
      at DateTime.toZone (datetime.ts:721:12)
      at DateTime.add (datetime.ts:811:65)
```

Working on it now.

**Comment by rogierschouten** [comment]
Fixed in version 1.27.2.  Thanks @xlegs !
